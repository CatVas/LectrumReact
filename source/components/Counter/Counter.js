
import { number } from 'prop-types';
import React from 'react';
import Styles from './styles.m.css';

const Counter = ({ count }) => (
    <section className = { Styles.counter }>
        Posts count: {count}
    </section>
);

Counter.defaultProps = {
    count: 0,
};
Counter.propTypes = {
    count: number,
};

export default Counter;
