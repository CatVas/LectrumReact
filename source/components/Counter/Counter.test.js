
import React from 'react';
import renderer from 'react-test-renderer';
import Counter from './';

const renderTree = renderer.create(<Counter count = { 3 } />).toJSON();

test('counter should match snapshot', () => {
    expect(renderTree).toMatchSnapshot();
});
