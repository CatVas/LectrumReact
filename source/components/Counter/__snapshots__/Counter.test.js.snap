// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter should match snapshot 1`] = `
<section
  className="counter"
>
  Posts count: 
  3
</section>
`;
